flocker/node/script.py

```diff
@@ -81,7 +82,7 @@ def _context_factory(path, host, port):
     node_credential = NodeCredential.from_path(path, b"node")
     policy = ControlServicePolicy(
         ca_certificate=ca, client_credential=node_credential.credential)
-    return policy.creatorForNetloc(host, port)
+    return (policy.creatorForNetloc(host, port), node_credential)
```

Comment:
Ick.  Tuple return type.  Even `namedtuple` would be better (so callers could use attribute lookup if they don't want to turn into a maintenance burden later).